packages/components/pie-toast-provider/test/component/pie-toast-provider.spec.ts

```diff
@@ -1,5 +1,10 @@
 import { test, expect } from '@sand4rt/experimental-ct-web';
 import { PieToastProvider } from '../../src/index.ts';
+import {
```

Comment:
Would it be possible to follow the new approach like we discussed earlier? Doing so will mean we have one less component to refactor 🙏🏼